### PR TITLE
Update home/pi/xilinx pages and minor 

### PR DESCRIPTION
**Pull request by zprood**
## Done

- Update  home page(/index.html)
- Update iot download page(/download/iot)
- Update raspberry pi download page(/download/raspberry-pi)
- Update server page(/download/server) with release schedule chart 
- Create xilinx download page(/download/xilinx)
- A few text alignments adjusting

### CODE REVIEW

File: templates/download/raspberry-pi.html [hunk 1]
```diff
@@ -200,114 +333,203 @@ <h6 class="u-text-light">
...
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h6 class="u-text-light">
+        <em>
+          * 仅提供32位
+        </em>
+      </h6>
+    <div>
```
Comment (anthonydillon) [inline templates/download/raspberry-pi.html:531]: This should be closing right?

File: templates/index.html [hunk 2]
```diff
@@ -18,6 +18,42 @@ <h4 id="takeover-subtitle">Raspberry Pi的Ubuntu桌面，最新的桌面功能
   </div>
 </section>
 
+<section class="p-strip u-no-padding--bottom">
+  <div class="row">
+    <div class="col-6 col-medium-4">
+      <h2>现代化的企业与开源</h2>
+      <p class="p-heading--four">Ubuntu的发布者。<br /> 安全，支持。代管服务。</p>
+      <p>
+        <a class="p-button--positive js-invoke-modal" href="/contact">
```
Comment (anthonydillon) [inline templates/index.html:27]:
```suggestion
        <a class="p-button--positive" href="/contact">
```

File: templates/index.html [hunk 3]
```diff
@@ -74,69 +110,1285 @@ <h3 class="p-heading--four"><a href="/engage/developer-desktop.html">开发者
...
+        </li>
+      </ul>
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+          <a class="p-link--external" href="https://ubuntu.com/robotics">
+            了解更多
+          </a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/contact" class="js-invoke-modal">
```
Comment (anthonydillon) [inline templates/index.html:1204]:
```suggestion
          <a href="/contact">
```

File: templates/server/index.html [hunk 4]
```diff
@@ -246,115 +246,7 @@ <h3 class="p-muted-heading u-align--center">可与您的硬件和软件一起使
...
-        </tr>
-      </tbody>
-    </table>
-  </div>
-</div>
-</div>
+{% include "shared/_release_schedule.html" %}
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/6.3.1/d3.min.js"></script>
 <script src="https://ubuntu.com/static/js/build/release-chart.min.js"></script>
```
Comment (anthonydillon) [inline templates/server/index.html:252]: This script is 404'ing

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Great work @zprood few small requests in the code.

File: templates/download/raspberry-pi.html (hunk 1 above)
Comment (zprood) [reply templates/download/raspberry-pi.html]: Yes, added a "/"

File: templates/index.html (hunk 2 above)
Comment (zprood) [reply templates/index.html]: Removed all the js-invoke-modal classes because it hasn't been used.

File: templates/index.html (hunk 3 above)
Comment (zprood) [reply templates/index.html]: Removed

File: templates/server/index.html (hunk 4 above)
Comment (zprood) [reply templates/server/index.html]: Replaced with the right URL.

**Review by anthonydillon — APPROVED** [review]
Great, thanks :+1:

**Pull request merged**